app/models/driver.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+class Driver < ApplicationRecord
+  # after_initialize :set_availability
+  has_many :trips
+  validates :name, presence: true
+  validates :vin, presence: true
+
+  def average_rating
+    ratings = trips.map {|trip| trip.rating}
+    if ratings = 0
+      return "NIL"
+    end
+    total_ratings = 0
+    ratings.each do |rating|
+      total_ratings += rating
+    end
+    average = total_ratings.to_f / ratings.length
+    return average.round(1)
```

Comment:
You can simplify this with `sum`:

```suggestion
    average = ratings.sum.to_f / ratings.length
    return average.round(1)
```